README.md

```diff
@@ -56,7 +56,7 @@ This will build journal using [browserify](http://browserify.org/) and will watc
 The `dev` script is enough to deploy, and requires node with Express installed. Otherwise, any other web server is perfectly fine -- when built, journal is simply a collection of files to be served over HTTP.
 
 #### Style Modifications
-Overriding CSS can be placed in the CSS style sheet located in [custom.css](./web/custom.css). To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html). Alternatively, the main style sheet [style.css](./web/style.css) can be edited.
+Overriding CSS can be placed in the CSS style sheet located in [style.css](./web/style.css). To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html). Alternatively, the main style sheet [style.css](./web/style.css) can be edited.
```

Comment:
Actually this is wrong; it should say "To modify the style of the app, the CSS in style.css can be changed."  no extra steps need to be taken, so you can remove the bit that says

>To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html).  Alternatively, the main style sheet [style.css](./web/style.css) can be edited.

Reply:
are you saying 

```
Overriding CSS can be placed in the CSS style sheet located in [style.css](./web/style.css). To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html). Alternatively, the main style sheet [style.css](./web/style.css) can be edited.
```

should be replace with

```
To modify the style of the app, the CSS in [style.css](./web/style.css) can be changed.
```

Reply:
yep :)